Src/BootCamp.Chapter/Items/ShoulderPiece.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter.Items
+{
+    public class Shoulderpiece
+    {
+        private string _name;
+        private decimal _price;
```

Comment:
Codacy found an issue: [Make '_price' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)